products/azul-zulu.md

```diff
@@ -40,6 +40,24 @@ identifiers:
 # - STS : EOL + 6 months
 # - custom for Java 6/7
 releases:
+-   releaseCycle: "24"
+    releaseDate: 2025-03-18 # https://docs.azul.com/core/release/24-ga/release-notes/release-notes
+    eol: 2025-09-16
+    eoes: 2025-09-16
+    latest: "24.28.83"
+    latestJdkVersion: "24.0.0+36"
+    latestReleaseDate: 2025-03-18
+    link: https://docs.azul.com/core/release/24-ga/release-notes/release-notes
+
+-   releaseCycle: "23"
+    releaseDate: 2024-09-17 # https://docs.azul.com/core/release/23-ga/release-notes/release-notes
+    eol: 2025-03-18
+    eoes: 2025-03-18
+    latest: "23.32.11"
+    latestJdkVersion: "23.0.2+7"
+    latestReleaseDate: 2024-07-16
```

Comment:
Looks like there is a mistake here because even its release date is newer than this date (2024-09-17 )

Reply:
so whether date of releaseDate or latestReleaseDate or both of them are wrong